_includes/binary.html

```diff
@@ -1,22 +1,21 @@
 <div id="container">
     <div id="binary">
-        <div class="column"><div class="column_heading">128</div><div id="7c" class="bit" onClick="toggle_bitc(7);">0</div></div>
-        <div class="column"><div class="column_heading">64</div><div id="6c" class="bit" onClick="toggle_bitc(6);">0</div></div>
-        <div class="column"><div class="column_heading">32</div><div id="5c" class="bit" onClick="toggle_bitc(5);">0</div></div>
-        <div class="column"><div class="column_heading">16</div><div id="4c" class="bit" onClick="toggle_bitc(4);">0</div></div>
-        <div class="column"><div class="column_heading">8</div><div id="3c" class="bit" onClick="toggle_bitc(3);">0</div></div>
-        <div class="column"><div class="column_heading">4</div><div id="2c" class="bit" onClick="toggle_bitc(2);">0</div></div>
-        <div class="column"><div class="column_heading">2</div><div id="1c" class="bit" onClick="toggle_bitc(1);">0</div></div>
-        <div class="column"><div class="column_heading">1</div><div id="0c" class="bit" onClick="toggle_bitc(0);">0</div></div>
+        <div class="column"><div id="7c" class="bit" onClick="toggle_bitc(7);">0</div><div class="column_heading">128</div></div>
+        <div class="column"><div id="6c" class="bit" onClick="toggle_bitc(6);">0</div><div class="column_heading">64</div></div>
+        <div class="column"><div id="5c" class="bit" onClick="toggle_bitc(5);">0</div><div class="column_heading">32</div></div>
+        <div class="column"><div id="4c" class="bit" onClick="toggle_bitc(4);">0</div><div class="column_heading">16</div></div>
+        <div class="column"><div id="3c" class="bit" onClick="toggle_bitc(3);">0</div><div class="column_heading">8</div></div>
+        <div class="column"><div id="2c" class="bit" onClick="toggle_bitc(2);">0</div><div class="column_heading">4</div></div>
+        <div class="column"><div id="1c" class="bit" onClick="toggle_bitc(1);">0</div><div class="column_heading">2</div></div>
+        <div class="column"><div id="0c" class="bit" onClick="toggle_bitc(0);">0</div><div class="column_heading">1</div></div>
         <div id="decimal1">= 0</div>
     </div>
 </div>
+
 <style>
     #binary			{width: 100%;}
-    #decimal1		{font-family: Arial, Helvetica, sans-serif; float: left; font-size: 5vw; width: 21vw; margin: 2.7vw 0 0; float: left}
+    #decimal1		{font-family: Arial, Helvetica, sans-serif; float: right; font-size: 4vw; width: 21vw; margin: 0.5vw -3.0vw 0; float: right}
```

Comment:
```suggestion
    #decimal1		{font-family: Arial, Helvetica, sans-serif; float: right; font-size: 4vw; width: 21vw; margin: 0.5vw -3.0vw 0; float: right}
```

You have actually hardcoded the #decimal1. It wont solve the problem.Earlier too it was hardcoded but the error occured only in case of rarest screen dimention.
<img width="557" alt="Screenshot 2019-07-18 at 2 41 08 am" src="https://user-images.githubusercontent.com/20012612/61412101-91597380-a905-11e9-8c41-d858fcf803ea.png">
<img width="658" alt="Screenshot 2019-07-18 at 2 47 14 am" src="https://user-images.githubusercontent.com/20012612/61412558-cc0fdb80-a906-11e9-909e-b1846765fb35.png">